### PR TITLE
Added support for Common Lisp implementation CLASP.

### PR DESCRIPTION
**Pull request by dg1sbg**
Please review and provide feedback. Thanks.
// Frank

### CODE REVIEW

File: fad.lisp [hunk 1]
```diff
@@ -309,6 +308,7 @@ might be removed instead!  This is currently fixed for SBCL and CCL."
                            #+:openmcl (cl-fad-ccl:delete-directory file)
                            #+:cormanlisp (win32:delete-directory file)
                            #+:ecl (si:rmdir file)
+                           #+:clasp (core:rmdir)
```
Comment (stassats) [inline fad.lisp]: Missing argument.

File: fad.lisp [hunk 2]
```diff
@@ -149,7 +148,7 @@ directory is returned as if by PATHNAME-AS-DIRECTORY."
                      directory-form)))
                (ignore-errors
                  (probe-file (pathname-as-file pathspec))))
-  #-(or :sbcl :cmu :scl :lispworks :openmcl :allegro :clisp :cormanlisp :ecl :abcl :digitool)
+  #-(or :sbcl :cmu :scl :lispworks :openmcl :allegro :clisp :cormanlisp :ecl :abcl :digitool :sbcl :clasp)
```
Comment (stassats) [inline fad.lisp]: Adding sbcl.

File: fad.lisp [hunk 3]
```diff
@@ -97,13 +97,12 @@ and CCL."
...
     (error "Can only list concrete directory names."))
-  #+:ecl
+  #+(or :ecl :clasp)
   (let ((dir (pathname-as-directory dirname)))
     (concatenate 'list
                  (directory (merge-pathnames (pathname "*/") dir))
                  (directory (merge-pathnames (pathname "*.*") dir))))
-  #-:ecl
-  (let ((wildcard (directory-wildcard dirname)))
+    #-(or :ecl :clasp) (let ((wildcard (directory-wildcard dirname)))
```
Comment (stassats) [inline fad.lisp]: Missing a newline and indentation.

Comment (dg1sbg) [inline fad.lisp]: will change and repost

File: fad.lisp (hunk 2 above)
Comment (dg1sbg) [inline fad.lisp]: :sbcl already in there  - at beginning of this line/statement. I only wanted to add :clasp here.

File: fad.lisp (hunk 1 above)
Comment (dg1sbg) [inline fad.lisp]: Will fix and repost.

**Comment by dg1sbg** [comment]
Please re-rewiew. I committed the fixes to my master branch.

**Comment by stassats** [comment]
Ok, but that :sbcl repetition is still there. And can you make it all into a single commit without mentioning "review" or anything like that, so that the history is nice and tidy.

**Comment by stassats** [comment]
Pushed after adjusting.

**Pull request closed without merging**